benchmarks/src/main/java/com/yahoo/omid/benchmarks/utils/Generator.java

```diff
@@ -15,7 +15,7 @@
  * LICENSE file.                                                                                                                                                                   
```

Comment:
This class was taken by someone at some point from the YCSB benchmark project. They have a different license header for that. I don't know if we can modify them or not according to the license :-(